site/app/templates/submission/RainbowGrades.twig

```diff
@@ -1,11 +1,32 @@
 <div class="content">
-    <h1 class="label">
-    {% if extra_label is defined %}
-        Rainbow Grades {{extra_label}}
-    {% else %}
-        Rainbow Grades
-    {% endif %}
-    </h1>
+    <header>
+        <h1 class="label">
+            {% if extra_label is defined %}
+                Rainbow Grades {{extra_label}}
+            {% else %}
+                Rainbow Grades
+            {% endif %}
+        </h1>
+        <div id="rainbow-grades-header">
+            {% if show_summary and extra_label is defined and extra_label == "For All Students" %}
```

Comment:
This check against a string to be shown in the UI feels like a hack.  Why not create a variable specifically for the "all students" case?